dags/run_all_processors.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+from airflow.models import DAG
+from airflow.operators.latest_only_operator import LatestOnlyOperator
+import utils.helpers as helpers
```

Comment:
The conventions are muddy. I prefer importing as `import foo`, as I think it's easier to read the code without having to guess where something comes from. However, Airflow itself uses `from foo import bar` and, as we followed their examples, we ended up doing that as well.

I don't see a need to change this specific case, but it would be good to use a single style. Probably something that'll wait until we have a linter that also checks the code style.